test/addFreakTest.js

```diff
@@ -0,0 +1,88 @@
+const { expect, assert } = require("chai");
+const EventEmitter = require('events');
+
+describe("FreakTeam", function () {
+  it("AddFreak fails when it's not called by hr", async function () {
```

Comment:
You can create another `describe` block, to specify the method that you are testing .

For example, wrap all your `it`s inside a 
```
describe("addFreak", async() => {
... //all tests
});
```

Add describe blocks for everytime you need a new context around a suite of tests.